src/reducers/match.js

```diff
@@ -51,7 +51,12 @@ export default (state = initialState, action) => {
 export const getMatch = state => state.app.match;
 export const getMatchData = state => state.app.match.match;
 export const getMatchLoading = state => state.app.match.loading;
-export const getMatchPlayers = state => state.app.match.match.players;
+
+export const getMatchPlayers = (state, team = '') =>
```

Comment:
Don't move the entire function, but move the logic dealing with team filtering.  I think the selectors here should just get and return pieces of state and not have any further logic than that.